public/download.php

```diff
@@ -1,8 +1,10 @@
 <?php
-require_once '../src/functions.php';
+if (!$_SERVER['REMOTE_ADDR'] === '::1') {
```

Comment:
Diese Änderungen sind glaube ich nicht notwendig, wenn man die _config.php jeweils lokal bei sich in config.php umbenennt. 🙂